docs/xostor/xostor.md

```diff
@@ -786,6 +786,77 @@ linstor sp list
 
  Use `linstor advise r` to see the commands to execute.
 
+### How can I delete an unreadable or unusable resource?
+
+In certain situations, a volume may no longer be usable.
+For example, a LINSTOR resource containing a VHD whose header/footer has been overwritten and is unreadable. Here, we're talking about a situation where a VHD isn't just simply corrupted and where `vhd-util repair -n <PATH>` is useless.
+For example we can have this output in `SMlog`:
+```
+Jun 22 10:50:13 r620-s3 SM: [23871] ['/usr/bin/vhd-util', 'query', '--debug', '-vsfpu', '-n', '/dev/drbd/by-res/xcp-volume-83da35c4-dd18-47fb-9d2b-68bd5b92fcaa/0']
+Jun 22 10:50:13 r620-s3 SM: [23871] FAILED in util.pread: (rc 22) stdout: 'error opening /dev/drbd/by-res/xcp-volume-83da35c4-dd18-47fb-9d2b-68bd5b92fcaa/0: -22
+```
+
+The problem with this error is that it's generic and can occur in other situations. If you're not sure what you're doing, contact us on support or the forum. Alternatively you can confirm that a resource is indeed unusable, execute this command by connecting to a host where the volume is marked `InUse`:
+```
+vhd-util check -n <DRBD_PATH>
+```
+In this example `<DRBD_PATH>` is equal to `/dev/drbd/by-res/xcp-volume-83da35c4-dd18-47fb-9d2b-68bd5b92fcaa/0`
+If the volume is not marked as `InUse` in the LINSTOR database, you can run this same command on any host that has the resource path on its filesystem.
+
+If the volume is indeed unusable and prevents an SR PBD-plug command, any action, or cannot be deleted via xe or XO, you can follow the instructions below.
+
+:::warning
+Again, if you're unsure of the situation, the procedure below is risky. It's indeed very rare to need to read this section of the documentation. There is only one major case where we consider that's useful to run these commands: A program like `dd` was executed on a resource, which destroyed the VHD headers/footers. Another similar scenario is a deletion of the replicas followed by a recreation of the resources.
+
+Additionally we assume that if you destroy a resource, you have a recent backup of the corresponding VM or VDI and you want to restore it if the data is important.
+:::
+
+1. Retrieve the SR's storage pool name using this command:
+```
+linstor sp list
+```
+
+Example output where the `group-name` is `xcp-sr-linstor_group_thin_device` in the `StoragePool` column:
```

Comment:
`group-name` is only used here in this part of the doc and could complicate the understanding of what to retrieve